caom2-search-server/examples/default/src/main/webapp/META-INF/context.xml

```diff
@@ -19,9 +19,17 @@
             validationQuery="select 123"
   />
 
-  <!-- development server -->
-  <!--<ResourceLink name="jdbc/uws"-->
-                <!--global="jdbc/postgresql-uws-9"-->
-                <!--type="javax.sql.DataSource"/>-->
+  <!-- standard dev/prod resource definition. ${} values are in <tomcat>/conf/catalina.properties -->
+<!--  <Resource name="jdbc/uws"-->
+<!--            auth="Container"-->
+<!--            type="javax.sql.DataSource"-->
+<!--            factory="org.apache.tomcat.jdbc.pool.DataSourceFactory" closeMethod="close"-->
+<!--            minEvictableIdleTimeMillis="60000" timeBetweenEvictionRunsMillis="30000"-->
+<!--            maxWait="20000"-->
+<!--            initialSize="0" minIdle="0" maxIdle="${search.uws.maxActive}" maxActive="${search.uws.maxActive}"-->
+<!--            username="${search.uws.username}" password="${search.uws.password}"-->
+<!--            driverClassName="org.postgresql.Driver" url="${search.uws.url}"-->
+<!--            removeAbandoned="false"-->
+<!--            testOnBorrow="true" validationQuery="select 123" />-->
```

Comment:
Please remove the commented one and add the parameterized values in (`${search.uws.xxx}).  That way it's completely configurable.  In the process, please rename them to `caom2search.uws.xxx`.